src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/OnboardingSupervisor.kt

```diff
@@ -137,6 +141,17 @@ internal class OnboardingSupervisor(
         }
     }
 
+    private fun retrieveChainRpcEndpoints() {
+        val url = "${helper.deploymentUri}/configs/rpc.json"
+        helper.get(url) { _, response, _, _ ->
+            if (response != null) {
+                Json.decodeFromString<Map<String, RpcInfo>>(response).let {
```

Comment:
need a try-catch here: `decodeFromString` throws `SerializationException` and `IllegalArgumentException`

Reply:
please resolve thread if it looks good after 894b4f2515986d3c5be0b3cda66234a6917b4328